lib/src/model/hearthstone_card.dart

```diff
@@ -0,0 +1,47 @@
+import 'package:flutter/cupertino.dart';
+
+class HearthstoneCard {
+  final String name;
+  final String cardId;
+  final String cardSet;
+  final String type;
+  final String playerClass;
+  final String? text;
+  final int? health;
+  final int? cost;
+  final int? attack;
+  final String locale;
+  final List<dynamic>? mechanics;
+  late final AssetImage image;
+
+  HearthstoneCard({
+    required this.cardId,
+    required this.cardSet,
+    required this.playerClass,
+    required this.name,
+    required this.type,
+    this.text,
+    this.health,
+    this.cost,
+    this.attack,
+    required this.locale,
+    this.mechanics,
+    String img = "assets/images/EX1_572.png",
```

Comment:
Add this asset route to constant file

Reply:
created a constant with the name "defaultImage"